### ISSUE TITLE
GUI updates

### ISSUE THREAD

**Issue by ajsdexgithub** [StephenBlackWasAlreadyTaken/xDrip-Experimental#229]
I was wondering if there were any plans to update/match the GUI's of xDrip and Nightwatch.

As an example, the statistics page in Nightwatch is better than the statistics page in xDrip. (Button interface)

There is also a center weighted 4x1 widget available in Nightwatch that is not available in xDrip.

My dream/wish is that visually, other than the app name, and the various settings required by xDrip, that the two apps look/feel/function equally.

Thanks for the time/cconsideration!

**Comment by tzachi-dar** [comment]
I have the same dream.
I also have a version of nightwatch that is an xDrip clone running on my
phone, still needs a little twisting to be publicly available. Currently it
is called nightwatchpro.
It should be renamed soon to xDripViewer.

So, either wait a few more weeks, or if you are brave, I can try and share
my version with you...

Thanks
Tzachi

**Comment by JohanDegraeve** [comment]
xDripViewer, will it have a lower minimum Android version ? (no BLE support needed)

**Comment by tzachi-dar** [comment]
4.2.2 is what I'm targeting it for, might run on lower versions, but I
don't have a way to try it.

**Comment by ajsdexgithub** [comment]
Thanks Tzachi!

I am running xdrip experimental on one phone...and Nightwatchpro on another, while using Dexcoms share DB.

It wasn't until I was playing with the settings in Nightwatchpro that I even noticed the differences.

I am happy to wait for those to merge/update...and happier to know that I'm not the only person interested in it.

Thanks again,
Adam

**Comment by JohanDegraeve** [comment]
sounds good !

**Comment by tzachi-dar** [comment]
Hi, one thing to note is that I'm testing it using the nighscout rest api,
but I don't think this should be a problem.

**Comment by JohanDegraeve** [comment]
is there any other option ?

**Comment by tzachi-dar** [comment]
If I understand him correctly, he is using a dexcom share to capture the data. 
See http://www.dexcom.com/dexcom-g4-platinum-share

**Comment by ajsdexgithub** [comment]
Correct.  I have the g4 platinum, but dont have any apple devices, so I use xDrip/Nightwatch and android phones to send/receive the data to/from the dexcom share server.
